### ISSUE TITLE
`r_thumb_add` loose joint

### ISSUE THREAD

**Issue by SimoneMic** [robotology/icub-tech-support#1632]
### Robot Name 🤖

ergoCub 1.0 S/N:000

### Request/Failure description

The joint is loose and doesn't stay fixed in place. The motor is not actuated.

![thumb1](https://github.com/robotology/icub-tech-support/assets/86918431/37464e83-91f8-4b4b-84fc-b3857901c2ce)

### Detailed context

![issue_hw](https://github.com/robotology/icub-tech-support/assets/86918431/97ffd8b2-6c2d-47ce-9fff-f0169d6f3875)

### Additional context

Probably, during the various calibrations and turn-offs, the thumb collides with the upper-leg cover and it got damaged.

**Comment by sgiraz** [comment]
Hi @SimoneMic, 

From the picture posted above It seems that a pin is coming out:


![Image](https://github.com/robotology/icub-tech-support/assets/18039045/ccfcbac1-6cbb-4b41-a682-dd1e22396be7)

Additionally, please keep in mind that we are already aware of the necessity to replace the right forearm with a new one in order to solve many other problems.

@GiacomoVassallo13 could you take a look at it?

cc @fbiggi @AntonioConsilvio

**Comment by fbiggi** [comment]
replacement of part code 16674 A1
gluing with Loctite 648 engine
MAXON_B7E7C938897D
replacement of the 2x6 pin on the support C_028_P_004 with screw m 2x8 + washer m2 + nut m2
movement test performed compliant
recovery time 2.5 hours



![20230908_090413](https://github.com/robotology/icub-tech-support/assets/20420331/89e744d1-5bf5-45c8-a306-65c86b2e8e79)
![20230908_074548(0)](https://github.com/robotology/icub-tech-support/assets/20420331/84589471-21a5-4df4-b338-be519e179d31)

@AntonioConsilvio @sgiraz @maggia80 @Fabrizio69

**Comment by maggia80** [comment]
the connector of the FAP of the thumb is damaged: 
![image](https://github.com/robotology/icub-tech-support/assets/6987330/ad15bbc7-7b57-4a04-8663-c654f381aee5)

this is due to the male connector that is damaged as well:

![image](https://github.com/robotology/icub-tech-support/assets/6987330/cdc11ea4-8dee-4fbb-90e8-50327487e43e)

we should pay extreme attention to these cables/connectors during the insertion. If they are damaged we should replace them.

I guess the MMA of the right palm is damaged. We should verify it and replace it.

**Comment by SimoneMic** [comment]
The adduction seems fine now, but there is another loose pin that doesn't keep the rest of the finger stiff:
![thumb3](https://github.com/robotology/icub-tech-support/assets/86918431/40aab0d1-431a-416d-a053-f23ff12afc4a)
![thumb4](https://github.com/robotology/icub-tech-support/assets/86918431/b49a8c27-b0f8-427b-b98a-4dd3c1979431)



Also I think that there's a missing spacer like this one (which was on the ground today):

![thumb5](https://github.com/robotology/icub-tech-support/assets/86918431/531f331e-7f5f-4475-b031-760157a2c26c)

**Comment by EhsanRanjbari** [comment]
Ciao @AntonioConsilvio, What is the latest situation with the thumb? Is it possible to fix it anytime soon? (I reserved the robot today from 11 a.m. and I need the right hand to do some experiments), thanks

**Comment by AntonioConsilvio** [comment]
Hi @EhsanRanjbari @SimoneMic, The phalange of the right thumb is broken:

![Broken](https://github.com/robotology/icub-tech-support/assets/114915464/ab1d599b-8276-4ebf-9568-6eb26c9d8ad4)


This is how it should be

![Undamaged](https://github.com/robotology/icub-tech-support/assets/114915464/b5cdc4ff-39b9-4dc9-8796-edb6679fe154)


As you can see in this photo, the part of the phalanx where the hole was to connect the phalanx to the pivot is broken.
>
> ![Screw](https://github.com/robotology/icub-tech-support/assets/114915464/9b69edec-89e1-4047-a977-e3f6739f36d3)




So we should replace this phalange with a new one.

cc @maggia80 @sgiraz @Fabrizio69 @fiorisi

**Comment by GiacomoVassallo13** [comment]
Hi. The broken component cod 16624.A3 hands thumb medial phalanx, hase been replaced. For this operation the finger was separated from the forearm.All the dowel pins were replaced and the new ones glued with Loctite648. 
The joint has been rewired due to disassembly of the thumb finger.
To rewire the joint it was necessary to separate the forearm from the robot, because the sheath had detached from the support from the side of the pulley.
The joint was tested with jogger set up before the arm was reattached.

cc: @Fabrizio69 @sgiraz @AntonioConsilvio

**Comment by AntonioConsilvio** [comment]
Hi, After replacing the damaged phalanx by @GiacomoVassallo13, I solved the electronic problems (see [comment](https://github.com/robotology/icub-tech-support/issues/1632#issuecomment-1711787645)) by replacing the MMA board and some wires.

Following these interventions, only one problem remains, namely that the thumb_add manages to close (0-->90), but not to reopen (90-->0).

The problem seems to be caused by the mechanism which, following the interventions described in this [comment](https://github.com/robotology/icub-tech-support/issues/1632#issuecomment-1711273002), has become very hard. 

In fact it has been tested with the jogger and the mechanism works correctly, but with yarpmotorgui it does not.

cc @maggia80 @sgiraz @Fabrizio69

**Comment by sgiraz** [comment]
Hi @SimoneMic,

Can you confirm that the joint involved in this issue is now working correctly?

If yes, I would close this issue.

**Comment by SimoneMic** [comment]
Yep! The joint moves from the motorgui, and is not loose anymore.

Thanks
